CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+Version XXX:
+
+* updated root certificates in examples
```

Comment:
This should have gone into the other commit

Reply:
agree